run-tests.sh

```diff
@@ -1,53 +1,56 @@
 #!/usr/bin/env bash
 
-# NOTE: set TEST_SUITE to be markers you want to run.
+# Tested on debian and alpine
+# You will need to install some dependencies yourself.
 
 set -eo pipefail
 
 export PYTHONIOENCODING="utf-8"
 export LANG=C.UTF-8
-export PIP_PROCESS_DEPENDENCY_LINKS="1"
+
 # Let's use a temporary cache directory
 export PIPENV_CACHE_DIR=`mktemp -d 2>/dev/null || mktemp -d -t 'pipenv_cache'`
 
-prefix() {
-	sed "s/^/   $1:    /"
-}
+# on some Linux OS python is python3
```

Comment:
Do we need to account for the fact that its python3 on some platforms?

Reply:
It is already accounted. You can set PYTHON:
```
PYTHON=${PYTHON:-"python"}
```
Usage example:
```
PYTHON=python3 PIPENV_PYTHON=3.9 ./run-tests.sh
```

Reply:
Also this piece of code wasn't used anywhere in the previous version before my change.